### DISCUSSION TITLE
CarPlay in knob/physical control mode on DS7 Crossback

### DISCUSSION THREAD

**Discussion by rybnico** [ludwig-v/wireless-carplay-dongle-reverse-engineering#73] [Q&A]
Hello,

I am using the dongle with the latest official firmware with my DS7 Crossback, a car from PSA/Stellantis that uses NAC entertainment system (like Peugeot/Citroen/Open).

CarPlay - unlike using it with the original wired CarPlay - is in "knob mode", meaning each button has a grey field around it, and sometimes the hardware controls on my steering wheel change the focused field in CarPlay while also controlling NAC in parallel.

Is this a problem on all PSA vehicles? Or is there a map of manufacturers with/without hardware control in the FW and DS is missing? Or how do they detect presence of hardware controls?

Thanks in advance for your help
Nico

**Comment by ludwig-v** [comment]
It is normal, CarPlay wireless = no transparency to save battery (even if the phone is charging)

**Reply by rybnico** [reply]
Hmm, okay. The visual change doesn't bother me but it is annoying that the hardware controls sometimes move the focus in CarPlay, e.g. if I change the radio station on the built-in entertainment.

**Reply by Somberland** [reply]
Not sure about the "knob mode" in my Peugeot3008 (bought 2020), but the "focus move" of buttons is not reproducible for me
